Classes/Domain/Model/Organisator.php

```diff
@@ -44,6 +44,13 @@ class Organisator extends \TYPO3\CMS\Extbase\DomainObject\AbstractEntity
      */
     protected $phone = '';
 
+    /**
+     * URL of Organisator
+     *
+     * @var string
+     */
+    protected $www = '';
```

Comment:
Please rename to `$link`

Reply:
I named this analogous to tt_address

Reply:
I know ext:tt_address uses `www`, but I think this is not very clear and possibly also called `www` due to historical reasons of the extension. 

See for example the definition of a person on https://schema.org/Person - there also `url` is used instead of `www`. So IMO either `link` (used 2 times in ext:sf_event_mgt) or `url` suites best here.

Reply:
ok

Reply:
I renamed www to link and pushed the commit to my repo.